examples/service-clients/azure-client/todo-list/src/app.tsx

```diff
@@ -14,7 +14,8 @@ import {
 	// TODO: Fix this once the tree-only mode API has been stabilized.
 	// eslint-disable-next-line import/no-internal-modules
 } from "@fluidframework/fluid-static/internal";
-import { createChildLogger } from "@fluidframework/telemetry-utils/legacy";
+// eslint-disable-next-line import/no-internal-modules
+import { createChildLogger } from "@fluidframework/telemetry-utils/internal";
```

Comment:
Note: we shouldn't need to change any imports in this PR. If we're forced to do that, it means that items that previously existed under `/legacy` are no longer there, which is a breaking change.